src/db/options.rs

```diff
@@ -312,3 +312,35 @@ pub struct ChangeStreamPreAndPostImages {
     /// If `true`, change streams will be able to include pre- and post-images.
     pub enabled: bool,
 }
+
+/// Determines the lifetime of a cursor.
+#[derive(Clone, Debug)]
+#[non_exhaustive]
+pub enum TimeoutMode {
+    /// Times out after one Iteration.
+    Iteration,
+    /// Times out after the end of the Cursor Lifetime.
+    CursorLifetime,
+}
+
+/// Specifies the options to a
+/// [`Database::RunCursorCommand`](../struct.Database.html#method.run_cursor_command) operation.
+#[derive(Clone, Debug, Default, TypedBuilder)]
+#[builder(field_defaults(default, setter(into)))]
+#[non_exhaustive]
+pub struct RunCursorCommandOptions {
+    /// Optional string enum value, one of 'iteration' | 'cursorLifetime'.
+    pub timeout_mode: Option<TimeoutMode>,
+    /// The default read preference for operations.
+    pub read_preference: Option<ReadPreference>,
+    /// Optional string enum value, one of 'tailable' | 'tailableAwait' | 'nonTailable'.
```

Comment:
```suggestion
    /// The type of cursor to return.
```
I think we can make this a little simpler -- the fact that this option is serialized as a string in the command is more of an implementation detail, and the possible options are already documented by the enum. (This is the doc comment for the `cursor_type` field on `FindOptions`.)